benchmarks/src/main/java/io/druid/benchmark/query/GroupByBenchmark.java

```diff
@@ -237,25 +250,69 @@ public void setup() throws IOException
       qIndexes.add(qIndex);
     }
 
-    OffheapBufferPool bufferPool = new OffheapBufferPool(250000000, Integer.MAX_VALUE);
-    OffheapBufferPool bufferPool2 = new OffheapBufferPool(250000000, Integer.MAX_VALUE);
-    final GroupByQueryConfig config = new GroupByQueryConfig();
+    StupidPool<ByteBuffer> bufferPool = new StupidPool<>(
+        new OffheapBufferGenerator("compute", 250000000),
```

Comment:
Can these be more descriptive, and are those values required to be fixed? Are the changes in direct memory needserves documented?